src/os/bluestore/BlueStore.cc

```diff
@@ -18208,7 +18207,19 @@ bool RocksDBBlueFSVolumeSelector::compare(BlueFSVolumeSelector* other) {
 
 static const std::string allocator_dir    = "ALLOCATOR_NCB_DIR";
 static const std::string allocator_file   = "ALLOCATOR_NCB_FILE";
-static uint32_t    s_format_version = 0x01; // support future changes to allocator-map file
+static uint32_t    s_format_version = 0x02; // support future changes to allocator-map file
```

Comment:
This is making mess.
Leave s_format_version = 0x01 as the offset-size version.
Create new bitmap_format_version = 0x02 as the new one.